test/controllers/orders_controller_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
```

Comment:
All the code of the orders controller shouldn't be here, this is orders_controller_TEST.rb, it should be in orders_controller.rb

Reply:
done